src/v2/view-builder/views/phone/ChallengeAuthenticatorDataPhoneView.js

```diff
@@ -33,10 +33,10 @@ const Body = BaseForm.extend(
       const sendText = ( this.model.get('primaryMode') === 'sms' )
         ? loc('oie.phone.verify.sms.sendText', 'login')
         : loc('oie.phone.verify.call.sendText', 'login');
+      const strongClass = this.model.get('phoneNumber') !== loc('oie.phone.alternate.title', 'login') ? 'strong' : '';
```

Comment:
instead of empty string i would add a class that doesn't make it strong.

Reply:
I couldn't find a class like that in courage.. What i see is that either the strong style is applied through the class
```
strong {
  font-weight: 600;
}
```
Or the class is not there which results in the font-weight being the same as the rest of the surrounding text

Reply:
or we add the class to the dom only when it's needed, otherwise default is always no bold.

Reply:
I think this is good, but can we rename the variable name? Maybe something like `extraCssClasses`?

Reply:
@mauriciocastillosilva-okta why adding an empty string is good? leaving DOM like this
`<span class>`

Reply:
let me take another look at this

Reply:
didn't see the other message before, but yes, my vote will be for not adding an empty class attribute if we don't need to. Agree 100% on that
I think that is what Gowthami implemented in the latest changes